src/test/javascript/spec/component/modeling-editor/modeling-editor.component.spec.ts

```diff
@@ -225,4 +226,47 @@ describe('ModelingEditorComponent', () => {
         const spanText = statusHint.query(By.css('span'))?.nativeElement?.textContent;
         expect(spanText).toBe('Saving...');
     });
+
+    it('should handle explanation input change', () => {
+        fixture.detectChanges();
+        const spy = jest.spyOn(fixture.componentInstance.explanationChange, 'emit');
+
+        const newExplanation = 'New Explanation';
+        fixture.componentInstance.onExplanationInput(newExplanation);
```

Comment:
Consider declaring a `component` variable that is representing the modeling editor component instead of accessing it via the fixture.

Reply:
Done.